src/deriver/api.py

```diff
@@ -13,6 +13,11 @@
 from .fragment_index import frag_index
 from .mate import mate
 import os
+import numpy as np
+from . import jensen_crossover as co
```

Comment:
variable names are too short and uninformative

Reply:
also this import is not quite right... 

`from .jensen_crossover import method1, method2`